docs/docs/basil.md

```diff
@@ -91,7 +91,7 @@ Now, to get the niceties that ZeppelinOS provides, let's install the `zos` comma
 
 ```sh
 npm install zos
-zos init basil 0.0.1
+zos init Basil 0.0.1
```

Comment:
We are using all lowercase for app names, please see https://docs.zeppelinos.org/docs/setup.html where `my-app` is used.

Reply:
Done, see latest commit.